app/src/main/java/com/x8bit/bitwarden/ui/platform/util/ConfigurationExtensions.kt

```diff
@@ -0,0 +1,15 @@
+@file:OmitFromCoverage
+
+package com.x8bit.bitwarden.ui.platform.util
+
+import android.content.res.Configuration
+import com.x8bit.bitwarden.data.platform.annotation.OmitFromCoverage
+
+/**
+ * A helper method to indicate if the current UI configuration is portrait or not.
+ */
+val Configuration.isPortrait: Boolean
+    get() = when (this.orientation) {
+        Configuration.ORIENTATION_LANDSCAPE -> false
```

Comment:
Isn't there a `Configuration.ORIENTATION_UNDEFINED`? I'm just curious how that would come into play here.

Reply:
I was matching existing logic where we treat `UNDEFINED` and `SQUARE` as portrait.

Reply:
OK sounds good.